docs/source/gettingstarted.rst

```diff
@@ -39,6 +39,11 @@ is to enable the NeuroDebian_ repository, and to install datalad as a system
 package::
 
   sudo apt-get install datalad
+  
+A current version of git-annex (as also provided by the NeuroDebian_ 
+repository) can be installed by typing::
+
+  sudo apt-get install git-annex
```

Comment:
yeah -- we need up to date one... should be `git-annex-standalone` from neurodebian